app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -10,3 +10,4 @@
 <%= presenter.attribute_to_html(:related_url, render_as: :external_link) %>
 <%= presenter.attribute_to_html(:resource_type, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:source) %>
+<%= presenter.attribute_to_html(:rights_statement) %>
```

Comment:
It renders the URI for the rights statement. So something like `https://rightstatements.org/2.0/blah` and that links out to the actual copyrights page. I was looking into the renderers and was wondering if thats how you would like it implemented. With that being said, im not going to have time to work on that architecture. If you all want those to be put in place, maybe it could be made into a ticket as an enhancement?